server/clients/stellar/transfer.go

```diff
@@ -192,3 +230,19 @@ func (c *Client) AwaitForTransactionWithMemoOnTfchainBridge(ctx context.Context,
 	}
 	return c.AwaitTransactionWithMemo(ctx, bridgeAddress, memo, timeout)
 }
+
+func (c *Client) Transactions(account string, limit uint, includeFailed bool, cursor string) ([]horizon.Transaction, error) {
```

Comment:
This is not only for transfers so it does not belong in `transfer.go`

Reply:
Is account.go a better place?